safenode/src/node/api.rs

```diff
@@ -403,7 +403,8 @@ impl Node {
     /// Retrieve a `Spend` from the closest peers
     async fn get_spend(&self, address: DbcAddress) -> Result<SignedSpend> {
         let request = Request::Query(Query::Spend(SpendQuery::GetDbcSpend(address)));
-        let responses = self.network.node_send_to_closest(&request).await?;
+        // NB: for discovery, might need query for the closest instead of local.
+        let responses = self.network.node_send_to_local_closest(&request).await?;
```

Comment:
Agreed, I think that for DBC related stuff we need to query to prevent attacks.
Here using `node_send_to_queried_closest` sounds safer.

Reply:
I saw this one! And changed it just now.

It actually _has_ to have query because getting parents is from across the whole network.

Reply:
Broadcast of valid spend and double spend attempted are using locally known closest though.